### ISSUE TITLE
Header and footer issue 

### ISSUE THREAD

**Issue by shannonbaldwin** [shannonbaldwin/pro.homeowner.ai#2]
@codextiny Need to remove header and footer off of[ this web page ](https://pro.homeowner.ai/opportunities-trailer/)but not disrupt global headers 

Please let us know how to do it and what code you are using so we know for next time.

support@milestones.ai 
M!l3st0nesL@bs

**Comment by codextiny** [comment]
@shannonbaldwin 

![image](https://github.com/user-attachments/assets/16460273-c07e-4280-868a-a8faad9ff7f5)

its done from our side, you just need to "quick edit" the page and change template type from "default template" to "Elementor Canvas" the header/Footer will be hidden.

Thanks,

**Comment by shannonbaldwin** [comment]
@codextiny Do I need to update this every time I make changes? It seems like I have to do that because if not, it will show up again.

**Comment by codextiny** [comment]
@shannonbaldwin 

no, no need to update page settings, its just need to do one time one time .
